### ISSUE TITLE
crashes on OSx

### ISSUE THREAD

**Issue by Ewan-Roberts** [trehn/termtrack#6]
command: 
`termtrack -figmntxo 1 iss`

Computer Details:

OS: macOS Mojave
Terminal Program: Kitty
Python: 3.6.5

stack trace:
```
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.6/bin/termtrack", line 10, in <module>
    sys.exit(main())
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/termtrack/cli.py", line 403, in main
    curses.wrapper(render, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/curses/__init__.py", line 94, in wrapper
    return func(stdscr, *args, **kwds)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/termtrack/utils/curses.py", line 429, in wrapper
    func(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/termtrack/cli.py", line 146, in render
    tle_file=tle,
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/termtrack/satellite.py", line 86, in __init__
    tle = TLE_REGEX.search(raw_html).group(1).strip().split("\n")
AttributeError: 'NoneType' object has no attribute 'group'
```

**Comment by GideonWolfe** [comment]
I am also seeing this crash on Arch Linux. Only when I actually attempt to view a satellite though, I can view the map fine otherwise

**Comment by trehn** [comment]
fixed in 0.7.3, thanks
